timed_release/connectors/mysql.py

```diff
@@ -0,0 +1,68 @@
+"""MySQL Connector.
+
+Manages interactions with MySQL
+for ows_timed_release_schedule database.
+"""
+
+from contextlib import contextmanager
+import functools
+from oto import response
+from sqlalchemy import create_engine
+from sqlalchemy import exc
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from timed_release import config
+from timed_release.connectors import sentry
+
+
+db_engine = create_engine(config.RDS_DB_URL, poolclass=config.POOL_CLASS)
+db_session = sessionmaker(bind=db_engine)
+
+baseModel = declarative_base()
+
+
+@contextmanager
+def db_session():
```

Comment:
either change the name of this function or change this variable name to something else - db_session = sessionmaker(bind=db_engine)
. It is conflicting